platform_driver_ethercat:
  ros__parameters:
    config_file: "/home/marta/ros2_ws/marta_ws/src/platform_driver_ethercat/config/marta.yaml"
